src/views/product/Product.tsx

```diff
@@ -12,7 +12,7 @@ import { cartSelector } from "../../store/selectors";
 
 type Props = {
   productVariant: FetchVariantResponse | null;
-  // categories: Category[];
+  // categories: Category[]
   // currencies: Currency[];
   // category: CategoryNames;
   // currency: Currency;
```

Comment:
If this were the selected currency, would it aid in currency selection? Say you changed the currency from the dropdown at the nav, the price of the product is supposed to changed based on the currency selected. Will this help in realising that feature?

Reply:
It won't. As it is now, these comments are redundant. We can remove them and the application will work as expected.